File: INICIANTE/1117/main.c
#include <stdio.h>

double average() {
    double grade1, grade2;
    scanf("%lf", &grade1);
    while (grade1 < 0 || grade2 > 10) {
        printf("nota invalida\n");
        scanf("%lf", &grade1);
    }
    scanf("%lf", &grade2);
    while (grade2 < 0 || grade2 > 10) {
        printf("nota invalida\n");
        scanf("%lf", &grade2);
    }
    return (grade1 + grade2) / 2;
}

int main() {
    printf("media = %.2lf\n", average());
    return 0;
}

File: INICIANTE/1117/main.js
function average() {
    let grade1 = parseFloat(lines.shift());
    while (grade1 < 1 || grade1 > 10) {
        console.log('nota invalida');
        grade1 = parseFloat(lines.shift());
    }
    let grade2 = parseFloat(lines.shift());
    while (grade2 < 1 || grade2 > 10) {
        console.log('nota invalida');
        grade2 = parseFloat(lines.shift());
    }
    return (grade1 + grade2) / 2;
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
console.log(`media = ${average().toFixed(2)}`);
